### PR TITLE
Change bug_report from an issue template to an issue form

### PR DESCRIPTION
**Pull request by tmccombs**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      placeholder: paste the output of `fd --version` here
+    validations:
+      required: true
+  - type: textarea
+    id: os
+    attributes:
+      label: Which operating system / distribution are you on?
+      placeholder: |
+        Unix: paste the output of `uname -srm` and lsb_release -a` here.
+        Windows: please tel us your Windows version
```
Comment (sharkdp) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:36]: typo (probably not from your PR): "tell"

**Review by sharkdp — APPROVED** [review]
Cool. Didn't know this was possible.

**Comment by tmccombs** [comment]
I didn't either before today.

**Comment by tavianator** [comment]
Is there a way to preview this?

**Comment by tmccombs** [comment]
That is an excellent question, and something I was wondering too. Maybe i could merge it into the main branch on my fork, and open an issue there to preview it?

**Comment by tavianator** [comment]
^ Good idea

**Comment by tmccombs** [comment]
image

**Comment by tmccombs** [comment]
Had a couple errors, but it's working now.

**Pull request merged**